packages/frontend-plugin-api/package.json

```diff
@@ -38,9 +38,11 @@
     "react-router-dom": "6.0.0-beta.0 || ^6.3.0"
   },
   "dependencies": {
+    "@backstage/core-components": "workspace:^",
     "@backstage/core-plugin-api": "workspace:^",
     "@backstage/types": "workspace:^",
     "@backstage/version-bridge": "workspace:^",
+    "@material-ui/core": "^4.12.4",
```

Comment:
Could you add a big TODO in relevant places that we need to get rid of this dep in the future? We definitely do not want the core plugin APIs to depend on MUI in the long run

Reply:
Done ✅ : https://github.com/backstage/backstage/pull/20586/commits/66af240ea692929df76d8f817e96595d648c44d6